en/mkdocs.yml

```diff
@@ -1028,6 +1023,11 @@ nav:
         - Kubernetes Operators:
           - K8s API Operator:
               - K8s API Operator Overview: install-and-setup/setup/kubernetes-operators/k8s-api-operator/k8s-api-operator.md
```

Comment:
This structure will break the build. When there are child pages, the parent title cannot have a page of its own. Let's have it like this?

- K8s Operator:
  - Operator Overview: <path>
  - Deploy Integrations using K8s Operator: <path>
  - Integration Deployment Samples:
    - Sample 1.. : <path>
    - Sample 2...: <path>
    - Sample 3...: <path>